nncf/torch/quantization/quantize_model.py

```diff
@@ -256,3 +258,13 @@ def send_to_device(tensor):
     compressed_model.nncf.disable_dynamic_graph_building()
 
     return compressed_model
+
+
+def compress_weights(model: torch.nn.Module, compress_weights: bool = False) -> torch.nn.Module:
```

Comment:
Also, I found it confusing that `compress_weights` function can be launched with option `compress_weights=False`.
Seems like nothing will happen on that call.
I'd use something like this: `fake_quantize` or `save_int_data`.

Reply:
Done